qemu/tests/qemu_guest_agent.py

```diff
@@ -3423,7 +3425,14 @@ def _result_check(rsult_qga, rsult_guest):
             date_group = re.search(r"driverdate.*\:\s(\d{4})(\d{2})(\d{2})",
                                    driver_info_guest, re.I).groups()
             driver_date_guest = "-".join(date_group)
-            _result_check(driver_date, driver_date_guest)
+            if main_qga_ver == "5":
```

Comment:
Also apply to `5.0` and `5.1` ?

Reply:
the qga for windows has a little difference with rhel, the "5.2" is the newest rebase version on rhel5.2, so there are no 5.0/5.1 or other tags.

Reply:
this command "get-devices" is windows only, btw.

Reply:
> the qga for windows has a little difference with rhel, the "5.2" is the newest rebase version on rhel5.2, so there are no 5.0/5.1 or other tags.

`rhel5.2` you mean `qemu5.2`?

Reply:
Or `qga5.2`?

Reply:
sorry, qemu5.2, typed it wrong. the latest windows qga was rebased on qemu5.2.

Reply:
For 5.2 version, I think it should be confirmed, since qemu 6.0 is coming out, if qemu-ga version changes with the qemu version, `main_qga_ver == "5":` will not be used later.

Reply:
So `main_qga_ver >= "5"`?

Reply:
the previous version is 101.x.x, 87.xx.x so >=5 is not suitable. I still cannot think out a appropriate method.  Maybe we can limit it to ">=5 and <=10"

Reply:
> the previous version is 101.x.x, 87.xx.x so >=5 is not suitable. I still cannot think out a appropriate method. Maybe we can limit it to ">=5 and <=10"

Thanks @kimi1978wy It seems a sensible choice.

Reply:
@6-dehan pls change main_qga_ver == "5":  to  ">=5 and <=10"

Reply:
yes, I've confirmed with dev that we'll obey the qemu version to rebase windows-qemu-ga. so it's better to set it as wyu said.